packages/cloud_functions/cloud_functions/pubspec.yaml

```diff
@@ -33,6 +33,12 @@ flutter:
       ios:
         pluginClass: FLTFirebaseFunctionsPlugin
       macos:
-        pluginClass: FLTFirebaseFunctionsPlugin
+        # pluginClass: FLTFirebaseFunctionsPlugin
+        # Temporary for development of Desktop packages
+        default_package: firebase_core_desktop
+      linux:
+        default_package: firebase_core_desktop
+      windows:
+        default_package: firebase_core_desktop
```

Comment:
```suggestion
        default_package: flutterfire_functions_desktop
      linux:
        default_package: flutterfire_functions_desktop
      windows:
        default_package: flutterfire_functions_desktop
```

Reply:
`firebase_core_desktop` is correct, the dart package starts with `flutterfire `

Reply:
Shouldn't this be `firebase_functions_desktop` instead then - this is the functions package not core 😅

Reply:
ops, correct